integration/tests/cook/util.py

```diff
@@ -615,6 +634,8 @@ def submit_job(cook_url, pool=None, headers=None, **kwargs):
     """Create and submit a single job"""
     if headers is None:
         headers = {}
+    if 'name' not in kwargs:
+        kwargs['name']="default_job-"+get_caller()
```

Comment:
Since we've got this in three places, can we make it a helper function, e.g. `make_default_job_name`?

Reply:
Adding another call into the stack means get_caller would now see another frame on the stack and get the wrong result, making that function more.... special. That said, it makes sense to refactor out the prefix.

Reply:
:+1: